src/NovelRT/NovelRunner.cpp

```diff
@@ -13,6 +13,7 @@ namespace NovelRT {
     _novelDotNetRuntimeService(std::make_unique<DotNet::RuntimeService>()),
     _novelRenderer(std::make_unique<Graphics::RenderingService>(this)) {
     _novelWindowingService->initialiseWindow(displayNumber, windowTitle);
+    _stepTimer.initializeValues();
```

Comment:
Added a function to StepTimer to initialize the `_frequency` and `_lastCounter` values.